File: 5-Matemcatica/1028-Figurinhas-3/1028.py
for i in range(int(input())):    
    x=input().split()
    x1=int(x[0])
    x2=int(x[1])
    maix=0
    omai=max(x1,x2)
    omin=min(x1,x2)
    reso=0
    while omin !=0:
        omai, omin = omin, omai % omin
    print(omai)

File: 5-Matemcatica/1028-Figurinhas-3/1028.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        int repeat = int.Parse(Console.ReadLine());
        for(int i = 0; i<repeat; i++){
            string[] nuns=Console.ReadLine().Split(' ');
            int maior=Math.Max(int.Parse(nuns[0]), int.Parse(nuns[1]));
            int menor=Math.Min(int.Parse(nuns[0]), int.Parse(nuns[1]));
            
            int inter;
            while(menor != 0){
                inter = menor;
                menor = maior%menor;
                maior = inter;
    
            }
            Console.WriteLine(maior);
        }
    }
}